image-swipe.d.ts

```diff
@@ -27,6 +32,9 @@ export class ImageSwipe extends ScrollView {
 
     public ios: any; /* UIScrollView */
     public android: any; /* android.support.v4.view.ViewPager */
+
+    public nextPage(): void;
```

Comment:
Not sure if that works nicely with the way the prev and next views are loaded in iOS. I can give that a try.